flowgate-common/src/main/java/com/vmware/flowgate/common/MetricName.java

```diff
@@ -0,0 +1,17 @@
+package com.vmware.flowgate.common;
+
+public class MetricName {
+   public static final String TEMP = "TEMP";
+   public static final String HUMIDITY = "HUMIDITY";
+   public static final String AIRFLOW = "AIRFLOW";
+   public static final String AIRPRESSURE = "AIRPRESSURE";
+   public static final String CONTACTCLOSURE = "CONTACTCLOSURE";
+   public static final String SMOKE = "SMOKE";
+   public static final String WATER = "WATER";
+   public static final String VIBRATION = "VIBRATION";
+   public static final String VOLTAGE = "VOLTAGE";
+   public static final String ACTIVE_POWER = "ACTIVEPOWER";
+   public static final String APPARENT_POWER = "APPARENTPOWER";
+   public static final String CURRENT = "CURRENT";
+   public static final String FREE_CAPACITY = "FREECAPACITY";
+}
```

Comment:
Let's give a prefix like PDU_XXX if the metric is related to PDU.

I think we also have a group of Metrics for the Server, please also define them here.

Reply:
> Let's give a prefix like PDU_XXX if the metric is related to PDU.
> 
> I think we also have a group of Metrics for the Server, please also define them here.

Done